ui_framework/README.md

```diff
@@ -21,28 +21,67 @@ fully-tested the code is.
 
 See the documentation in [`scripts/jest.js`](../scripts/jest.js) for more options.
 
-### React components
-
-Here are the components you can import from the Framework:
-
-```javascript
-import {
-  KuiButton,
-  KuiButtonGroup,
-  KuiButtonIcon,
-} from '../path/to/ui_framework/components';
-```
-
 ## Creating components
 
 There are four steps to creating a new component:
 
 1. Create the SCSS for the component in `ui_framework/components`.
```

Comment:
Ah ignore that comment! I wasn't paying close enough attention. You're right, I'll fix this.